gh-actions/build-changelog/index.js

```diff
@@ -0,0 +1,31 @@
+const core = require('@actions/core');
+const { GitHub } = require('@actions/github');
+
+async function run() {
+  try {
+    const myToken = core.getInput('GITHUB_TOKEN');
+
+    const tag = core.getInput('tag_name', { required: true });
+    const label = tag.substr(10, tag.length - 1);
+
+    const query = `type:pr+label:${label}`;
+
+    const octokit = new GitHub(myToken);
+
+    const { data: pullRequest } = await octokit.search.issuesAndPullRequests({
```

Comment:
This github library seems pretty cool! Seems like there's lots of cools stuff we could do with it.